src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx

```diff
@@ -28,37 +28,49 @@ export class VanguardArtistWrapper extends React.Component<
     const { article } = this.props
     const { hero_section, layout, slug, title } = article
     const { isExpanded } = this.state
+    const textColor = isExpanded ? "white" : undefined
 
     return (
-      <ArtistContainer pb={4} maxWidth={1000} px={4} mx="auto">
-        <Box textAlign="center">
-          <ArtistTitle size="8">{title}</ArtistTitle>
-          <Box position="absolute">
-            <Share
-              // TODO: We may need to use custom urls for in-page routing
-              url={getFullEditorialHref(layout, slug)}
-              title={title}
-            />
+      <Box background={isExpanded && "black"}>
+        <ArtistContainer pb={4} maxWidth={1000} px={4} mx="auto">
+          <Box textAlign="center">
+            <ArtistTitle size="8" color={textColor}>
```

Comment:
Do we know how styled components will resolve an undefined color? Should we just default it to something?

Reply:
They will default to black or the system color